scilpy/reconst/fiber_coherence.py

```diff
@@ -6,10 +6,23 @@
 NB_FLIPS = 4
 ANGLE_TH = np.pi / 6.
 
+# directions to the 26 neighbors.
+# Preparing once rather than in compute_fiber_coherence, possibly called many
+# times.
+all_d = np.indices((3, 3, 3))
```

Comment:
possibly called many times, but generating 27 values is not very long. Anyway I don't mind having it there but I think it should be in capital letters like the other constant values.